ENH: Add macOS arm64 builds

matthewfeickert opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [N/A] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

matthewfeickert committed:
ENH: Enable macOS arm64 builds

* Add osx-arm64 support.
* Bump build number.

matthewfeickert commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

matthewfeickert commented:
In the `.configure --prefix=$PREFIX` step I'm hitting

```
looking for gcc... /Users/runner/miniforge3/conda-bld/looptools_1732598279384/_build_env/bin/arm64-apple-darwin20.0.0-clang
looking for g++... /Users/runner/miniforge3/conda-bld/looptools_1732598279384/_build_env/bin/arm64-apple-darwin20.0.0-clang++
looking for fortran... /Users/runner/miniforge3/conda-bld/looptools_1732598279384/_build_env/bin/arm64-apple-darwin20.0.0-gfortran
extracting the Fortran libraries... ok
does /Users/runner/miniforge3/conda-bld/looptools_1732598279384/_build_env/bin/arm64-apple-darwin20.0.0-gfortran append underscores... error linking Fortran and C
Traceback (most recent call last):
...
```

but I am unable to locally reproduce by running

```
python build-locally.py osx_arm64_
```

on an Apple silicon machine.

matthewfeickert commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/12190276573](https://github.com/conda-forge/conda-forge-webservices/actions/runs/12190276573). Examine the logs at this URL for more detail.</sub>

matthewfeickert commented:
@conda-forge-admin, please rerender

matthewfeickert committed:
MNT: Re-rendered with conda-smithy 3.61.2 and conda-forge-pinning 2026.04.30.15.52.02

Other tools:
- conda-build 26.3.0
- rattler-build 0.63.1
- rattler-build-conda-compat 1.4.14

matthewfeickert commented:
Seems that the original issues I had were an emulation problem, as native osx-arm64 builds pass fine.

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

I considered the following status checks when analyzing this PR:
 - **linter**: passed
 - **azure**: passed
 - **github-actions**: passed


Thus the PR was passing and merged! Have a great day!
